scripts/lib.mjs

```diff
@@ -0,0 +1,53 @@
+import fs from "fs"
```

Comment:
I hate these two js files. Too procedural. Please check out the following resources:
https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api
https://medium.com/poka-techblog/simplify-your-javascript-use-map-reduce-and-filter-bd02c593cc2d

Reply:
I have tried to make it less procedural. Thanks for the resources, I didn't know there is a promise-based filesystem API for node.js